# Homework #5
|Author| Stanley A. Baronett|
|--|-------------------------------|
|Created | 10/6/2020|
|Updated | 10/7/2020|

All references to chapter problems, numbered equations, and page numbers are from _Astrophysics in a Nutshell: Second Edition_ by Dan Maoz.

## Chapter 4, Problem 4

### a.

From the virial theorem, Eq. 4.43,

\begin{align}
\bar P V = -\tfrac{1}{3}E_\textrm{gr},
\end{align}

where $\bar P$ is the mean pressure,

\begin{align}
V = \frac{4\pi}{3}r^3,
\end{align}

is the volume (Eq. 4.46), and

\begin{align}
E_\textrm{gr} \sim -\frac{GM^2}{r},
\end{align}
is the characteristic gravitational self-energy.  For a white dwarf of mass $M$ and radius $R$, we find

\begin{equation}
\boxed{ \bar P \sim \frac{GM^2}{R^4} }.
\end{equation}

### b.

The speed of sound is $v_\textrm{s} = \sqrt{dP/d\rho} \sim \sqrt{P/\rho}$.  Assuming uniform density, $\rho \sim M/R^3$, an order-of-magnitude estimate of the sound speed is

\begin{align}
v_\textrm{s} \sim \sqrt{\frac{GM}{R}}.
\end{align}

In an accreting white dwarf with a carbon core that has reached nuclear ignition temperature, a nuclear burning "flame" encompasses the star at the sound velocity or faster.  Assuming $R = 10^4 \textrm{ km}$ and $M = 1.4M_\odot$, we can estimate the time it takes the flame to traverse the radius of the white dwarf as follows:

In [1]:
km = 1e5     # cm
R = 1e4*km   # cm
Msun = 2e33  # g
M = 1.4*Msun # g
G = 6.7e-8   # erg*cm/g^2
v_s = (G*M/R)**(1/2)
tau = R/v_s

print('𝑣ₛ ~ {:.2e} cm/s ~ {:.0f} km/s'.format(v_s, v_s/1e5))
print('𝜏ₛ ~ {:.2f} s'.format(tau))

𝑣ₛ ~ 4.33e+08 cm/s ~ 4331 km/s
𝜏ₛ ~ 2.31 s


\begin{equation}
\boxed{v_\textrm{s} \approx 4331 \textrm{ km/s}}.
\end{equation}

\begin{equation}
\boxed{\tau_\textrm{sc} \approx 2.31 \textrm{ s}}.
\end{equation}

Note that this __sound-crossing timescale__ is $\sim(G\rho)^{-1/2}$, which is also the free-fall timescale (Eq. 3.15).

### c.

If the entire mass of the white dwarf is synthesized from carbon to nickel, with a mass-to-energy conversion efficiency of 0.1%, then the total energy output is

\begin{equation}
E_\textrm{tot} = 0.1\% Mc^2
\end{equation}

We can calculate the energy of the explosion as follows:

In [2]:
c = 3e10 # cm/s
E_tot = 0.001*M*c**2

print('𝐸_tot = {:.2e} erg'.format(E_tot))

𝐸_tot = 2.52e+51 erg


\begin{equation}
\boxed{E_\textrm{tot} \approx 2.52 \times 10^{51} \textrm{ erg}}.
\end{equation}

From Eq. 4.82, we can calculate the gravitational binding energy of the white dwarf as follows:

In [3]:
E_gr = 5e53*(M/1.4/Msun)**2/(R/10e5)

print('𝐸_gr = {:.2e} erg'.format(E_gr))
print('𝐸_tot/𝐸_gr = {:.0f}'.format(E_tot/E_gr))

𝐸_gr = 5.00e+50 erg
𝐸_tot/𝐸_gr = 5


\begin{equation}
E_\textrm{gr} \approx 5 \times 10^{50} \textrm{ erg}.
\end{equation}

\begin{equation}
\boxed{ \frac{E_\textrm{tot}}{E_\textrm{gr}} \approx 5 }.
\end{equation}

Comparing the two, we see that __the explosion energy is about 5 times the gravitational binding energy__, and therefore the white dwarf is disrupted completely and leaves no remnant.

### d.

The optical luminosity of a type-Ia supernova are completely driven by gamma rays from the radioactive decays of $^{56}\textrm{Ni} \rightarrow \,^{56}\textrm{Co} + \gamma \rightarrow \,^{56}\textrm{Fe} + \gamma$.  The energy radiated per decay chain is

\begin{equation}
E_\gamma = [m(^{56}\textrm{Ni}) - m(^{56}\textrm{Fe})]c^2.
\end{equation}

If the atomic weights of $^{56}\textrm{Ni}$ and $^{56}\textrm{Fe}$ respectively are $55.942135$ and $55.934941$, then we we can calculate the total energy radiated in the optical range during the event as follows:

In [4]:
m56NiDa = 55.942135  # Da
m56FeDa = 55.934941  # Da
DaeV    = 931.5e6    # dalton in eV/c^2
eV      = 1.602e-12  # electron volt in erg

E_gamma = (m56NiDa - m56FeDa)*DaeV*eV # in ergs

Dag     = 1.66e-24    # dalton in g
m56Nig  = m56NiDa*Dag # mass of 56Ni in g
N       = M/m56Nig    # total number of nickel atoms

E_tot_rad = N*E_gamma

print('𝐸_tot-rad = {:.2e} erg'.format(E_tot_rad))

𝐸_tot-rad = 3.24e+50 erg


\begin{equation}
\boxed{E_{\textrm{tot}-\gamma} \approx 3.24 \times 10^{50} \textrm{ erg}}.
\end{equation}

Given the characteristic times for the two decay processes are 8.8 days and 111 days, we can calculate the typical luminosity as follows:

In [5]:
day  = 24*3600         # s
ct   = (8.8 + 111)*day # s
L    = E_tot_rad/ct    # erg/s
Lsun = 3.8e33          # erg/s

print('𝐿 ~ {:.2e} 𝐿sun'.format(L/Lsun))

𝐿 ~ 8.23e+09 𝐿sun


\begin{equation}
\boxed{\bar L \sim 10^{10} L_\odot}.
\end{equation}

### e.

The photon diffusion time to the surface at radius $R$ is $R^2/(lc)$, where the mean free path is $l = (\kappa\rho)^{-1}$.  If we express $\rho = M/V$, where $V = \tfrac{4}{3}\pi R^3$, and replace $R$ with $vt$, where $v$ is a constant expansion velocity, solving for $t$ we find

\begin{align}
t &= \frac{3v^2 t^2 \kappa M}{4\pi c v^3 t^3} \\
&= \sqrt{ \frac{3\kappa M}{4\pi c v} }.
\end{align}

If the shell has a mass $M = 1.4M_\odot$, $v = 3 \times 10^4 \textrm{ km/s}$, and an atomic-line opacity of $\kappa \approx 0.3 \textrm{ cm}^2\textrm{/g}$, we can calculate the time it takes for light to diffuse, in a random walk, through the expanding shell of gas as follows:

In [6]:
import numpy as np

v     = 3e4*km # cm/s
kappa = 0.3    # cm^2/g
t     = (3*kappa*M/4/np.pi/c/v)**(1/2) # s

print('𝑡 ~ {:.0f} days'.format(t/day))

𝑡 ~ 17 days


\begin{equation}
\boxed{t \sim 17 \textrm{ days}}.
\end{equation}

This corresponds to the _light-curve rise time_ for a type-Ia supernova (i.e, from explosion to reaching its maximum brightness), which is simply the time it takes for the radioactive decay luminosity from the entire shell to become visible.